inlang/source-code/paraglide/paraglide-js-adapter-sveltekit/README.md

```diff
@@ -191,14 +193,16 @@ You can also use parameters in your translations. This is useful for pages that
 
 ```js
 import { createI18n } from "@inlang/paraglide-js-adapter-sveltekit"
-import * as runtime from "../paraglide/runtime.js"
+import * as runtime from "$lib/paraglide/runtime.js"
 
 export const i18n = createI18n(runtime, {
-	"/user/[username]" : {
-		en: "/user/[username]",
-		de: "/benutzer/[username]",
-		fr: "/utilisateur/[username]",
-	},
+	pathnames: {
+		"/user/[username]" : {
```

Comment:
side note: but it's not clear from this documentation how `[param]` should be translated. I suppose it can be done with a custom function?

Reply:
`[param]` isn't translated by the adapter, it's just a placeholder